src/app/features/reporting/reporting/reporting.component.scss

```diff
@@ -11,3 +13,27 @@
 .view-section {
   margin-bottom: 1em;
 }
+
+.error-message {
```

Comment:
maybe we should create a generic "error-message" component, that generalized the layout of this?

Reply:
Yes, we should do this. I would go even further and introduce an application wide error notification system, something like a toast bar or a notification center. Let's discuss this later.

Reply:
See https://github.com/Aam-Digital/ndb-core/issues/2417